README.md

```diff
@@ -2,15 +2,16 @@ This bot adds initial information at the start of year for Wikimedia Sverige. It
 
 # Installation
 Install required libraries with pip:
-
-`$ pip install -r requirements.txt`
+```
```

Comment:
I prefer this implementation to that in #46. Could possibly add the `console` language tag though gor additional context